dynatrace-operator-google-marketplace/chart/dynatrace-operator/templates/configmap.yaml

```diff
@@ -471,9 +1406,68 @@ data:
       networkZone: {{ .Values.networkZone }}
       {{- end }}
 
-      {{- if .Values.activeGate.image }}
       activeGate:
+        {{- if .Values.activeGate.image}}
         image: {{ .Values.activeGate.image }}
+        {{- end}}
+        autoUpdate: {{ .Values.activeGate.autoUpdate }}
+
+      oneAgent:
+        {{- if .Values.oneAgent.image }}
+        image: {{ .Values.oneAgent.image }}
+        {{- end }}
+        autoUpdate: {{ .Values.oneAgent.autoUpdate }}
+        {{- if .Values.oneAgent.version }}
+        version: {{ .Values.oneAgent.version }}
+      {{- end }}
+
+      {{- if .Values.classicFullStack.enabled }}
```

Comment:
Nothing about routing, yet we have them in values.yaml

Reply:
removed routing from values and schema yaml